pkg/persistence/postgrespersister.go

```diff
@@ -114,20 +139,83 @@ func (p *PostgresPersister) PopulateBlockDataFromDB(tableName string) error {
 	return nil
 }
 
-// CreateTables creates tables in DB if they don't exist
-func (p *PostgresPersister) CreateTables() error {
-	eventTableQuery := postgres.CreateEventTableQuery()
+// CreateVersionTable creates the version table
```

Comment:
Should also note here in the doc that as a side effect `p.version` will get set in this method with the new or existing version value.  Or there could be a method "SetPersisterVersion(version *string)" with L150-161.